doc/ug/install_instructions/index.md

```diff
@@ -77,10 +77,10 @@ To build device software you need a baremetal RV32IMC compiler toolchain.
 You can either build your own or use a prebuilt one.
 We recommend installing the toolchain to `/tools/riscv`.
 
-#### Option 1 (recommended): Use the lowRISC-provided prebuilt GCC toolchain
+#### Option 1 (recommended): Use the lowRISC-provided prebuilt toolchain
 
-lowRISC provides a prebuilt GCC toolchain for the OpenTitan project.
-Download the file starting with `lowrisc-toolchain-gcc-rv32imc-` from [GitHub releases](https://github.com/lowRISC/lowrisc-toolchains/releases/latest) and unpack it to `/tools/riscv`.
+lowRISC provides a prebuilt toolchain for the OpenTitan project. This toolchain contains both GCC and Clang, targeting RISC-V. By default the device software is built with Clang.
+Download the file starting with `lowrisc-toolchain-rv32imc-` from [GitHub releases](https://github.com/lowRISC/lowrisc-toolchains/releases/latest) and unpack it to `/tools/riscv`.
 
 Or alternatively, use a in-tree helper script.
```

Comment:
Actually can we suggest using the in-tree helper script first? If you download to a non-standard location, the helper script makes updates to the configuration, which I don't want to have to walk people through doing on their own.